.evg.yml

```diff
@@ -129,9 +145,10 @@ tasks:
   - name: sbom
     commands:
       - func: "generate sbom"
+      # TODO: uncomment this before merging branch back into master, to ensure we still upload to silk from the main branch
       - func: "upload sbom"
-      - func: "push SBOM Lite to Silk"
-      - func: "pull augmented SBOM from Silk"
+      # - func: "push SBOM Lite to Silk"
+      # - func: "pull augmented SBOM from Silk"
       - func: "scan sbom"
 
   - name: ssdlc-artifacts-release
```

Comment:
I think you can get rid of that task completely.
The compliance report links to the augmented SBOM and static code analysis report which we won't have.